### ISSUE TITLE
Can't build on macOS or Debian

### ISSUE THREAD

**Issue by OptimusKoala** [Embarcadero/pa-radserver-docker#7]
Hi, 

I have this error when I build the image from macOS or Debian:

`> [19/39] RUN sed -i "s/uname -a | grep/awk -F= '\/^NAME\/\{print $2\}' \/etc\/os-release | grep/" /tmp/apachesetup.sh ; sed -i "s/uname -a | grep/awk -F= '\/^NAME\/\{print $2\}' \/etc\/os-release | grep/" /tmp/rssetup.sh ; sed -i "s/systemctl restart apache2.service/service apache2 restart/" /tmp/apachesetup.sh ; sed -i "s/systemctl status apache2.service --no-pager/service apache2 status/" /tmp/apachesetup.sh:
0.332 sed: can't read /tmp/apachesetup.sh: No such file or directory
0.350 sed: can't read /tmp/rssetup.sh: No such file or directory
0.369 sed: can't read /tmp/apachesetup.sh: No such file or directory
0.387 sed: can't read /tmp/apachesetup.sh: No such file or directory

  78 |     # as init system (PID 1)" bugs in the apachesetup.sh and rssetup.sh
  79 | >>> RUN sed -i "s/uname -a | grep/awk -F= '\/^NAME\/\{print \$2\}' \/etc\/os-release | grep/" /tmp/apachesetup.sh ; \
  80 | >>> sed -i "s/uname -a | grep/awk -F= '\/^NAME\/\{print \$2\}' \/etc\/os-release | grep/" /tmp/rssetup.sh ; \
  81 | >>> sed -i "s/systemctl restart apache2.service/service apache2 restart/" /tmp/apachesetup.sh ; \
  82 | >>> sed -i "s/systemctl status apache2.service --no-pager/service apache2 status/" /tmp/apachesetup.sh
  83 |

ERROR: failed to solve: process "/bin/sh -c sed -i \"s/uname -a | grep/awk -F= '\\/^NAME\\/\\{print \\$2\\}' \\/etc\\/os-release | grep/\" /tmp/apachesetup.sh ; sed -i \"s/uname -a | grep/awk -F= '\\/^NAME\\/\\{print \\$2\\}' \\/etc\\/os-release | grep/\" /tmp/rssetup.sh ; sed -i \"s/systemctl restart apache2.service/service apache2 restart/\" /tmp/apachesetup.sh ; sed -i \"s/systemctl status apache2.service --no-pager/service apache2 status/\" /tmp/apachesetup.sh" did not complete successfully: exit code: 2`

**Comment by lmbelo** [comment]
Is your Mac equipped with an ARM processor?

**Comment by OptimusKoala** [comment]
Yes, but I've made it for amd64 and I also tried on a debian server. Same error.

**Comment by lmbelo** [comment]
Are you virtualizing the Debian server in your ARM Mac? If you're using parallels, try to install the x64 Ubuntu image they provide, it works fine, but don't forget to set the x64 arch in your scripts.

**Comment by OptimusKoala** [comment]
I did not virtualized, I tried on a debian server, just to ensure that it's wasn't about the macOS.

It's seems that two files are missing  -> /tmp/apachesetup.sh and /tmp/rssetup.sh 
I can't find them or find how to create them.

**Comment by lmbelo** [comment]
Is it possible for you to test on Ubuntu?
